server/src/models/internshipModule.ts

```diff
@@ -239,6 +213,37 @@ InternshipModuleSchema.methods.submitCompleteDocumentsPdf = async function (
   return this.save();
 };
 
+/*********************/
+/* Model Event Hooks */
+/*********************/
+async function trySetPassed(document: Document) {
+  if (
+    document.get("status") === InternshipModuleStatuses.PLANNED &&
+    document.get("aepPassed") === true &&
+    (await isWeeksTotalLongEnough(document.get("internships"))) &&
+    document
+      .get("internships")
```

Comment:
Funktioniert das so? Als ich das neulich ausprobiert hab, da war `internships` nur ne Liste an ObjectIds, bis ich `await this.populate("internships").execPopulate();` aufgerufen hab, damit die auch geladen werden.

Reply:
Hmmm... vlt geht es auch nicht. Aber sowieso stimmt hier irgendwas nicht - aber ich weiß nicht was. Der Status wird jedenfalls nicht zu passed geändert. Ich komme mit dem Debugger nicht so richtig klar. Vielleicht kannst du nochmal gucken?

Reply:
Fixed in 6719ff

Reply:
Fixed in 6719fffe544950f95793c9f4ea8a9931bceb33c1